src/lua/init.lua

```diff
@@ -44,6 +44,26 @@ tarantool_exit(int);
 
 local fio = require("fio")
 
+local function sourcepath()
```

Comment:
I think `path` is unclear in whether it is a path to a file or to a directory.

Reply:
Did it in #4260